packages/cli-tools/src/resolveTransitiveDeps.ts

```diff
@@ -0,0 +1,421 @@
+import fs from 'fs-extra';
+import path from 'path';
+import * as fetch from 'npm-registry-fetch';
```

Comment:
this is unexpected, since node 18 there is global `fetch`. let's just call it `npmRegistryFetch`